docs/level_0/first_example_component.md

```diff
@@ -0,0 +1,136 @@
+# First Example Component
+
+In this example we describe how to develop the component that we have used in the
+[First Example Installation](./first_example_installation.md).
+
+The component contains a Component Descriptor and a Blueprint with one DeployItem. The DeployItem specifies how to 
+deploy the nginx as a helm chart.
+
+## Deploy Item
+
+We want to deploy an nginx via the helm chart [nginx ingress](https://github.com/kubernetes/ingress-nginx/tree/master/charts/ingress-nginx).
+Therefore, we specify a DeployItem in a yaml file with the following content.
+
+```yaml
+deployItems:
+- name: nginx
+  type: landscaper.gardener.cloud/helm                                                             # (1)
+  target:
+    name: {{ index .imports "target-cluster" "metadata" "name" }}                                  # (2)
+    namespace: {{ index .imports "target-cluster" "metadata" "namespace" }}                        # (3)
+  config:
+    apiVersion: helm.deployer.landscaper.gardener.cloud/v1alpha1
+    kind: ProviderConfiguration
+
+    chart:
+      ref: "eu.gcr.io/gardener-project/landscaper/tutorials/charts/ingress-nginx:v0.1.0"           # (4)
+
+    updateStrategy: patch
+
+    name: nginx                                                                                    # (5)
+    namespace: first-example                                                                       # (6)
+```
+
+Since we want to deploy a helm chart, the type of the DeployItem is set to `landscaper.gardener.cloud/helm` in line (1).
```

Comment:
```suggestion
Since we want to deploy a helm chart, the type of the DeployItem is set to `landscaper.gardener.cloud/helm` in line (1). This line specifies which deployer will later pick up the DeployItem.
```